app/src/main/java/io/neurolab/main/output/audio/PlaybackInfoListener.java

```diff
@@ -0,0 +1,51 @@
+package io.neurolab.main.output.audio;
+
+// An abstract class providing some methods(to be implemented) for updating the user interface when media player evokes the corresponding events.
+public abstract class PlaybackInfoListener {
+
+    // Presently we need only 5 basic states. We will add the remaining states as required in the future for our application.
+    static final int INVALID = -1;
```

Comment:
Best way to add props like these is using Java enums

Reply:
@CloudyPadmal Is it okay if I use @interface annotation for defining the states (like enums) for achieving the same purpose as there are abstract classes involved and also inner class usages.😕

Reply:
Yes, that's okay